gravitee-am-ui/package.json

```diff
@@ -38,7 +38,7 @@
     "@angular/router": "16.2.12",
     "@ctrl/ngx-codemirror": "^7.0.0",
     "@gravitee/ui-components": "3.45.2",
-    "@gravitee/ui-particles-angular": "11.0.1",
+    "@gravitee/ui-particles-angular": "12.0.1",
```

Comment:
Looks like the v12.0 is targeting Angular 17, in our 4.3.x branch, we are still in angular 16. is it an issue ?

![image](https://github.com/gravitee-io/gravitee-access-management/assets/3110552/7aebbf8e-e58b-4749-93c3-e8585217ed9e)

Reply:
yes, it may be, we should update the angular first

Reply:
updated to 11.3.1 instead